README.md

```diff
@@ -3,6 +3,13 @@ http-verbs-test
 
 [![Build Status](https://travis-ci.org/hmrc/http-verbs-test.svg)](https://travis-ci.org/hmrc/http-verbs-test) [ ![Download](https://api.bintray.com/packages/hmrc/releases/http-verbs-test/images/download.svg) ](https://bintray.com/hmrc/releases/http-verbs-test/_latestVersion)
 
+## This library is deprecated
+### All classes from this library have been migrated to [http-verbs](https://github.com/hmrc/http-verbs). This is available for Play 2.5, 2.6, and 2.7. Please replace with the following dependency:
```

Comment:
I think it's enough to tell people to look at `http-verbs`. `http-verbs` is where we should document how to use the test module.

Reply:
@colin-lamed Good point well made. Updated.